### PR TITLE
Example added

### PR DESCRIPTION
**Pull request by zbigniew-gajewski**
Stock portfolio optimization.

### CODE REVIEW

File: Flips.Examples/FlipsExampleStocks/Program.fs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    // Printing results:
+    printfn "-- Result --"
+    match results with
+    | Suboptimal msg -> printfn "Unable to solve. Error: %s" msg
+    | Optimal solution ->
+        printfn "Objective Value - Risk: %f %%" (solution.ObjectiveResult * 100.0)
+    
+        for (decision, value) in solution.DecisionResults |> Map.toSeq |> Seq.filter(fun (key, value) -> value > 0.0) do
+            let (DecisionName name) = decision.Name
+            printfn "Decision: %s\tValue: %f %%" name (value * 100.0)
```
Comment (matthewcrews) [inline Flips.Examples/FlipsExampleStocks/Program.fs:82]: I love this example but I'd like for it to be moved to its own file in the Flips.Examples project. Could you add this as a file named `StocksExample.fs` and include the contents of `Utils.fs` in it? I'm going to be breaking the examples out into their own files. I just haven't gotten to that yet.

You could include the README as a comment in the `StocksExample.fs` file

I would then add a reference to the example in the `Program.fs` file.

Comment (matthewcrews) [reply Flips.Examples/FlipsExampleStocks/Program.fs:82]: I have actually updated the layout of the Flips.Examples project. Each example has its own directory now. This should make it easier for you. Just add your own directory for your example, move your files into that directory, and add a call to run the example in the `Program.fs` file. You should be able to look at the other examples to see how this is done. I wanted to support your README.md file. By having you put your example in its own directory with a README.md you should see some nice documentation. Thanks for stoking that thought. I look forward to getting this merged!

Let me know if any of that is unclear.

Comment (zbigniew-gajewski) [reply Flips.Examples/FlipsExampleStocks/Program.fs]: Done.

**Comment by zbigniew-gajewski** [comment]
Refactored as suggested (moved to StocksExample directory, compressed into one file, added some packages to Flips.Examples.fsproj).

**Pull request merged**